client/state/form/shipping-label/actions.js

```diff
@@ -1,9 +1,13 @@
+import * as FormActions from 'state/form/actions';
+
 export const OPEN_PRINTING_FLOW = 'OPEN_PRINTING_FLOW';
 export const EXIT_PRINTING_FLOW = 'EXIT_PRINTING_FLOW';
 
-export const openPrintingFlow = () => ( {
-	type: OPEN_PRINTING_FLOW,
-} );
+export const openPrintingFlow = () => ( dispatch ) => {
+	dispatch( { type: OPEN_PRINTING_FLOW } );
+	dispatch( FormActions.setFormProperty( 'pristine', false ) );
+	dispatch( FormActions.nextStep() );
+};
```

Comment:
You want those to happen in order, right? It appears that the "redux way" to do this might be this way: https://github.com/tshelburne/redux-batched-actions